modules/util/portal-tools-rest-builder-test-impl/rest-openapi.yaml

```diff
@@ -196,6 +222,36 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/TestEntity"
             tags: ["TestEntity"]
+    /test-entities-address/{testEntityId}/test-entities-address:
```

Comment:
I am not sure if this test is reproducing the exact issue. You are returning a Page of EntityAdress, so IMHO the plural should be something like test-entity-addresses. To be more accurate, if you are referring to the TestEntity, the path should start with /test-entities/{testEntityId}/....

The use case that I see we are supporting from Commerce should be returning only one TestEntityAddress.

/test-entities-address/{testEntityId}/test-entity-address

I have updated the file to reproduce the exact issue. Please, execute rest builder and update the files according to the new java interfaces. You can merge all the buildREST commits with the existing one

Reply:
Thank you Carlos, totally agree! Your proposal sounds more "REST" and reproduces the exact issue :)